fortran/src/hdf5_fortrandll.def.in

```diff
@@ -76,6 +78,8 @@ H5D_mp_H5DGET_ACCESS_PLIST_F
 H5D_mp_H5DWRITE_PTR
 H5D_mp_H5DREAD_PTR
 H5D_mp_H5DVLEN_RECLAIM_F
+H5D_mp_H5DREAD_MULTI_F
+H5D_mp_H5DWRITE_MULTI_F
```

Comment:
Are these duplicate entries intentional?

Reply:
where are the duplicates?

Reply:
fixed